src/style.css

```diff
@@ -0,0 +1,154 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  font-family: "Roboto", sans-serif;
+  color: #000;
+}
+
+body {
+  font-size: 11px;
+  background-color: #fff;
+}
+
+[title] { /*a logo  */
+  text-decoration: none;
+  padding: 23px 0 22px;
+}
+
+.header-container {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  height: 60px;
+  padding: 0 50px;
+  background: #fff;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14px;
+  text-transform: uppercase;
+}
+
+.nav-list {
+  display: flex;
+  padding: 0;
+  list-style: none;
+}
+
+li.nav-element {
+  position: relative;
+  display: flex;
+  align-items: center;
+  margin-right: 20px;
+}
+
+.nav-element > a {
+  text-decoration: none;
+  /* display: flex; */
+  color: #000;
+}
+
+nav a:hover {
+  color: #00acdc;
+}
+
+a.is-active::after {
+  content: "";
+  display: block;
+  position: absolute;
+  margin-top: 18.5px;
+  background-color: #00acdc;
+  border-radius: 8px;
+  width: 100%;
+  height: 4px;
+  /* top: 0;
+  bottom: -30px; */
+}
+
+main {
+  margin: 50px 0;
+  background: #fff;
+}
+
+.cards-catalog {
+  margin: 0 auto 30px;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  width: 944px;
+}
+
+.foto img {
+  width: 158px;
+  margin: 30px 20px 43px;
+}
+
+div.productname p {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 15px;
+  margin: 0;
+}
+
+.card {
+  width: 200px;
+  height: 402px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  box-sizing: border-box;
+  border-radius: 5px;
+  margin-bottom: 50px;
+}
+
+.wrap {
+  padding: 0 15px;
+}
+
+div.productcode p {
+  margin: 6px 0 0;
```

Comment:
It's hard to read and not practical.
If you need only one side, write it. 
`margin-top: 6px;`